src/internal/types.ts

```diff
@@ -8,11 +8,12 @@ import {
   type KeysOfUnion,
   type Simplify,
   type Split,
+  type EmptyObject,
 } from "type-fest";
 
 declare const __brand: unique symbol;
 
-export type Branded<T, Brand extends symbol> = T & { [__brand]: Brand };
+export type Branded<T, Brand extends PropertyKey> = T & { [__brand]: Brand };
```

Comment:
Why the change? We want to limit the usage of branded types as much as possible, but this opens up the usage to regular strings...

Reply:
I see, I changed that to also support a string, as in the related test in `mapValues`. Maybe I could just change the type in the test from:
```ts
type UserID = Branded<string, "UserId">;
```
to
```ts
type UserID = Branded<string, symbol>;
```
and revert that change in `Branded` type.